subprojects/resources-http/src/main/java/org/gradle/internal/resource/transport/http/HttpClientConfigurer.java

```diff
@@ -159,6 +169,32 @@ public void configureUserAgent(HttpClientBuilder builder) {
         builder.setUserAgent(UriTextResource.getUserAgentString());
     }
 
+    private void configureCookieSpecRegistry(HttpClientBuilder builder) {
+        PublicSuffixMatcher publicSuffixMatcher = PublicSuffixMatcherLoader.getDefault();
```

Comment:
We can't use CookieSpecRegistries since this has been introduced in http client 4.5 and we are using 4.4.1 :(